http-common/src/proxy.rs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use futures::future::TryFutureExt;
+use std::{
+    future::Future,
+    io,
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use tokio::io::{AsyncRead, AsyncWrite, ReadBuf};
+
+#[cfg(feature = "tokio1")]
```

Comment:
You already have a `#[cfg(feature = "tokio1")]` block on the `mod proxy` declaration in `http-common/src/lib.rs`, so there's no reason to clutter up this file with a whole bunch of redundant `#[cfg(feature = "tokio1")]` declarations.

Reply:
Done.